annofabcli/annotation/change_annotation_properties.py

```diff
@@ -0,0 +1,286 @@
+        properties: AnnotationDetailForCli,
+        force: bool = False,
+        backup_dir: Optional[Path] = None,
+    ):
+        super().validate_project(project_id, [ProjectMemberRole.OWNER])
+
+        project_title = self.facade.get_project_title(project_id)
+        logger.info(f"プロジェクト'{project_title}'に対して、タスク{len(task_id_list)} 件のアノテーションのプロパティを変更します。")
+
+        if backup_dir is not None:
+            backup_dir.mkdir(exist_ok=True, parents=True)
+
+        for task_index, task_id in enumerate(task_id_list):
+            logger.info(f"{task_index+1} / {len(task_id_list)} 件目: タスク '{task_id}' のアノテーションのプロパティを変更します。")
+            self.change_properties_for_task(
+                project_id,
+                task_id,
+                annotation_query=annotation_query,
+                properties=properties,
+                force=force,
+                backup_dir=backup_dir,
+            )
+
+    def main(self):
+        args = self.args
+        project_id = args.project_id
+        task_id_list = annofabcli.common.cli.get_list_from_args(args.task_id)
+
+        dict_annotation_query = get_json_from_args(args.annotation_query)
+        annotation_query_for_cli = AnnotationQueryForCli.from_dict(dict_annotation_query)
+        try:
+            annotation_query = self.facade.to_annotation_query_from_cli(project_id, annotation_query_for_cli)
+        except ValueError as e:
+            print(f"{self.COMMON_MESSAGE} argument '--annotation_query' の値が不正です。{e}", file=sys.stderr)
+            sys.exit(COMMAND_LINE_ERROR_STATUS_CODE)
+
+        properties_of_dict = get_json_from_args(args.properties)
+        properties_for_cli = AnnotationDetailForCli.from_dict(properties_of_dict)
+
+        if args.backup is None:
+            print("間違えてアノテーションを変更してしまっときに復元できるようにするため、'--backup'でバックアップ用のディレクトリを指定することを推奨します。", file=sys.stderr)
+            if not self.confirm_processing("復元用のバックアップディレクトリが指定されていません。処理を続行しますか？"):
+                sys.exit(COMMAND_LINE_ERROR_STATUS_CODE)
+            backup_dir = None
+        else:
+            backup_dir = Path(args.backup)
+
+        self.change_annotation_properties(
+            project_id,
+            task_id_list,
+            annotation_query=annotation_query,
+            properties=properties_for_cli,
+            force=args.force,
+            backup_dir=backup_dir,
+        )
+
+
+def main(args):
+    service = build_annofabapi_resource_and_login(args)
+    facade = AnnofabApiFacade(service)
+    ChangePropertiesOfAnnotation(service, facade, args).main()
+
+
+def parse_args(parser: argparse.ArgumentParser):
+    argument_parser = ArgumentParser(parser)
+
+    argument_parser.add_project_id()
+    argument_parser.add_task_id()
+
+    EXAMPLE_ANNOTATION_QUERY = (
+        '{"label_name_en": "car", "attributes":[{"additional_data_definition_name_en": "occluded", "flag": true}]}'
+    )
+
+    parser.add_argument(
+        "-aq",
+        "--annotation_query",
+        type=str,
+        required=True,
```

Comment:
タスク内のすべてのアノテーションに対して、`is_pretected:true`を設定したいときがあるので、オプショナル引数にして欲しいです。
すみません、伝え忘れていました。


```suggestion
        required=False,
```

またオプショナル引数にしたことで、型ヒントなどの修正も必要になります。こちらも修正をお願いしたいです。